src/DependencyGraph/DependencyGraph.ts

```diff
@@ -125,15 +128,15 @@ export class DependencyGraph {
     }
 
     this.correctInfiniteRangesDependency(address)
+
+    return this.getAndClearContentChanges()
   }
 
-  public setCellEmpty(address: SimpleCellAddress) {
-    const vertex = this.addressMapping.getCell(address)
+  public setCellEmpty(address: SimpleCellAddress): ContentChanges {
+    const vertex = this.shrinkPossibleMatrixAndGetCell(address)
```

Comment:
what if vertex is already `null`, why do we then shrink matrix that contains it?